app/com/arpnetworking/kairos/service/KairosDbServiceImpl.java

```diff
@@ -262,15 +264,11 @@ private static MetricsQuery useAvailableRollups(
                     //
                     // For any given query, we can at best use the rollup based on the smallest
                     // sampling unit, if any.
```

Comment:
Can you remove / rewrite this comment block since it's no longer true?

I would opt for just removal since you've already explained what makes a unit "usable" in the helper method itself